adversary-intelligence/README.md

```diff
@@ -0,0 +1,136 @@
+|                       | pattern        | derived using `type` (ie:file-object:hashes.md5:838eb145bfa5c04a7077d43cd77642c5) |
+|                       | pattern_type   | "stix"                                                                            |
+|                       | spec_version   | "2.1"                                                                             |
+|                       | type           | "indicator"                                                                       |
+| published_date        | valid_from     |                                                                                   |
+|                       | confidence     | malicious_confidence: "high" = 80, "medium" = 60, "low" = 30, "other" = 10        |
+| kill_chain_phases     | killchain      |                                                                                   |
+| indicator_types       | threat_types   |                                                                                   |
+| label\.name           | labels         |                                                                                   |
+
+#### CrowdStrike Indicator types to STIX pattern
+
+| CrowdStrike Indicator Type | STIX Pattern              |
+|---------------------------|----------------------------|
+| hash_md5                  | file:hashes.MD5            |
+| hash_sha256               | file:hashes.'SHA-256'      |
+| hash_sha1                 | file:hashes.'SHA-1'        |
+| url                       | url:value                  |
+| domain                    | domain-name:value          |
+| ip_address                | ipv4-addr:value            |
+| mutex_name                | mutex:name                 |
+| password                  | user-account:credential    |
+| file_name                 | file:name                  |
+| email_address             | email-addr:value           |
+| username                  | user-account:account_login |
+| persona_name              | user-account:display_name  |
+| ip_address_block          | ipv4-addr:value            |
+| coin_address              | x-wallet-addr:value        |
+| bitcoin_address           | x-wallet-addr:value        |
+
+- Once all indicators have been mapped, they're uploaded to Sentinel Threat Intelligence using the [Threat Intel upload indicators API](https://learn.microsoft.com/en-us/azure/sentinel/upload-indicators-api).
+
+### Configuration
+
+The following parameters are required for the ARM template top be deployed.
+
+- FunctionName - A name to be given to the Azure Function App. Default is 'CSFalconIOC'
+- CrowdStrikeClientId - The client id of the credentials generated in the CrowdStrike console
+- CrowdStrikeClientSecret - The client secret of the credentials generated in the CrowdStrike console
+- CrowdStrikeBaseUrl - The base url of the region in which the CrowdStrike credentials were generated
```

Comment:
Not sure if this is something we control or not, but I would stick with our common naming schema:
FalconClientID

Reply:
Oof. We could, but the process to get that changed could take a while as the integration is already deployed to the marketplace.